simplewebrtc.js

```diff
@@ -231,6 +231,7 @@ function SimpleWebRTC(opts) {
         });
     });
     this.webrtc.on('localScreenStopped', function (stream) {
+        self.emit('localScreenRemoved');
```

Comment:
webrtc.on('videoRemoved', ...) only works for remote videos but not for local videos. 
An event that would be fired when a local video has been removed would be nice.